lib/matcher.js

```diff
@@ -1,3 +1,4 @@
+var _ = require('lodash');
 var PROPERTY_TO_FLAG_AS_STRUMMER_MATCHER = "IS-STRUMMER-MATCHER";
 
 function Matcher(opts) {
```

Comment:
Yes, we are handling, and the json schema will also removes the optional field from `required` array